SharedSources/VLCRemoteControlService.m

```diff
@@ -32,7 +32,8 @@ @implementation VLCRemoteControlService
     if (@available(iOS 9.1, *)) {
         [commands addObject:cc.changePlaybackPositionCommand];
     }
-    return commands;
+    NSArray *finalCommands = [commands copy];
+    return finalCommands;
```

Comment:
I think you should directly `return [commands copy];` without creating a variable for it.